### DISCUSSION TITLE
Cannot find module 'cypress/integration/some-file.json' or its corresponding type declarations

### DISCUSSION THREAD

**Discussion by coler-j** [cypress-io/cypress#23129] [Questions and Help]
**I am having trouble importing `json` into ts tests.**

My `tsconfig` looks like:

In `/cypress/tsconfig.json`
```
{
  "compilerOptions": {
    "allowSyntheticDefaultImports": true,
    "esModuleInterop": true,
    "resolveJsonModule": true,
    "moduleResolution": "node",
    "importHelpers": true,
    "allowJs": true,
    "target": "es5",
    "lib": ["es5", "dom"],
    "types": ["cypress", "node"]
  },
  "include": ["**/*.*"]
}
```

I have a test file like:
In `cypress/integration/sometest.ts`
```
import thing from "cypress/integration/some-file.json"

.... rest of code
```

Where `cypress/integration/some-file.json` is a valid json file.

I get the error:

> "Cannot find module 'cypress/integration/some-file.json' or its corresponding type declarations"

I have added `resolveJsonModule` to the tsconfig and restarted my IDE / TS server but am still having the issue.

I am on TS version 4.6.4
And ` "cypress": "10.3.0",`
as well as 
```
"@cypress/webpack-batteries-included-preprocessor": "2.2.3",
"@cypress/webpack-preprocessor": "5.11.0",
```

Do I need some special webpack configuration to recongnize json files?

**Comment by stavares843** [comment]
hello! 👋 

have you installed the `@types` of the module you want to use?

**Comment by coler-j** [comment]
I am just trying to import a JSON file from within my own repository.
